index.html

```diff
@@ -1528,6 +1528,34 @@ <h2>A project for first-time contributions</h2>
 
         </div>
         <!-- ________ Contributor's card END ________  -->
+
+          <!-- ________ Contributor's card START ________  -->
+          <div class="card">
+
+            <p class="name">Irene Tomaini</p>
+            <p class="contact">
+                <i class="fa fa-twitter"></i>
+                <a href="https://twitter.com/irenetomaini1" target="_blank">@irenetomaini1</a>
+            </p>
+            <p class="about">
+                I've been started to study JavaScript for being a frontend developer. I like cooking, cats and marvel superheroes!
+            </p>
+            <div class="resources">
+                <p>3 Useful Dev Resources</p>
+                <ul>
+                    <li>
+                        <a href="#" target="_blank" title="https://www.freecodecamp.org/">FreeCodeCamp</a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank" title="https://github.com/getify/You-Dont-Know-JS">You Don't Know JS</a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank" title="https://reactjs.org/docs/hello-world.html">React Docs</a>
```

Comment:
For the three links above you need to put the URL (https:....) Inside of `href` replacing he current #. The `title` attribute is for a short description.